Firebase/Core/FIRApp.m

```diff
@@ -133,23 +133,6 @@ + (void)configureDefaultAppWithOptions:(FIROptions *)options
       [FIRApp sendNotificationsToSDKs:sDefaultApp];
       sDefaultApp.alreadySentConfigureNotification = YES;
     }
-
-    if (![FIRAppEnvironmentUtil isFromAppStore]) {
```

Comment:
Why are we removing this? Don't we want to warn the users that this might not work?

Reply:
I'm fairly certain that the project won't even compile with a target below 8.0, so this message could never show. I can confirm that's the case, though.

Reply:
Xcode 7.3 won't compile at all, and I got a warning stating that Core has a minimum target of 8.0.